pkg/configurator/validating_webhook.go

```diff
@@ -319,7 +319,7 @@ func checkEnvoyLogLevels(configMapField, configMapValue string) bool {
 	return valid
 }
 
-// checkEnvoyLogLevels checks that the field value is a valid log level
+// checkEnvoyImage checks that the namespace and registry of the envoy proxy sidecar image are valid
```

Comment:
What does namespace imply?

Reply:
Updated to `name`. I am not sure namespace was the correct term

Reply:
Thanks, makes sense now.